script.js

```diff
@@ -0,0 +1,578 @@
+const KeyBoard = {
+
+  elements: {
+    title: null,
+    textAria: null,
+    keyBox: null,
+    keys: [
+      ['Backquote', 'Digit1', 'Digit2', 'Digit3', 'Digit4', 'Digit5', 'Digit6', 'Digit7', 'Digit8', 'Digit9', 'Digit0', 'Minus', 'Equal', 'BackSpace'],
```

Comment:
It is easier to read when code is formatted   
![image](https://user-images.githubusercontent.com/18613713/79037760-95889800-7bdc-11ea-8ca8-06456f6c7dcd.png)

Reply:
My prettier doesn't format arrays as you show. What tools are better to use?
Done

Reply:
:thumbsup: